app.js

```diff
@@ -1,88 +1,767 @@
 // Task 1 - Notes for JS Evaluation Predictions
 
+/* 
+1. 6
+2. -15
+3. 1
+4. 11
+5. false
+6. true
+7. false
+8. catdog
+9. 2pizza
+10. NaN
+11. false
+12. false // incorrect was true thought strict operator will would take in consideration the type Of Num
+13. true
+14. true
+15. false
+16. true
+17. false
+18. true
+19. false
+20. true
+21. true
+22. true
+23. true
+24. false
+25. 'n'
+26. 'e'
+27. 'a'
+28. 'I'
+29. undefined // indexOf returns -1 when index is not found not undefined
+30. 4
+31. true
+32. true
+
+*/
+
+
+
+
 // List 3 expressions that surprised you or gave unexpected results, why?
+// 29 i thought would returned undefined, now learned that if index is not found it always returns -1
+// 12 i thought since the strict operator was being used it would evaluate to false since one has no decimal point, maybe got confused because in other they would be different types such as float and int
```

Comment:
In JavaScript numbers are always stored as a `float` so even if you save a variable as `5` JS see's it as `5.0`. Some other coding languages will see `floats` and `integers` as different types